examples/app/index.html

```diff
@@ -1,21 +1,14 @@
 <!doctype html>
```

Comment:
why remove the explanatory comments??

I would not expect any change in this file just from introducing the closure binary rule.